velox/functions/prestosql/aggregates/ReduceAgg.cpp

```diff
@@ -297,7 +297,12 @@ class ReduceAgg : public exec::Aggregate {
     // Do not evaluate on null input.
     SelectivityVector remainingRows = rows;
     if (input->mayHaveNulls()) {
-      remainingRows.deselectNulls(input->rawNulls(), 0, rows.size());
+      DecodedVector decoded(*input, rows);
+      if (auto* rawNulls = decoded.nulls()) {
+        auto bits = remainingRows.asMutableRange().bits();
```

Comment:
@kagamiori Can we not use remainingRows.deselectNulls here?

Reply:
Updated. Thanks!